src/tests/server.js

```diff
@@ -35,10 +35,15 @@ describe('Server', function() {
 
       it('should throw an error for empty arguments', function(done) {
         // fix this test
-        request.post('http://127.0.0.1:8000/maps/submit', function (error, response, body) {
-          expect(error);
-          done();
-        });
+        handlers.getRoutes(null, null)
+          .then(function (response) {
+            expect(!response);
+            done();
+          })
```

Comment:
wouldn't there still be a response even if there are no arguments for this request?